### DISCUSSION TITLE
What python version are we targeting?

### DISCUSSION THREAD

**Discussion by wireboy5** [Slow-Scallion4183/Adventure-Colussus#26] [Q&A]
@Slow-Scallion4183  I Have a real quick question.

What python version are we targeting? I am currently testing on 3.9 and 3.8. All looks well on both of these versions.

Edit: A better question would be what python version are you using?

Another real quick question:
Would it be alright if I started implementing using pydantic for the Human class and clean up the character save function a bit?

Cheers, and Happy Coding! :)

**Comment by robotlightsyou** [comment]
I'm using 3.9, but I can go as far back as 3.6 if we need to for some reason(please don't take away my f strings).

edit: I vote yes to editing classes.

**Reply by Slow-Scallion4183** [reply]
i vote yes too :D

**Comment by ghost** [comment]
3.9.2 Latest = Best!

**Comment by Slow-Scallion4183** [comment]
Im also using Python 3.9 :D It would be fine if you start implementing classes using pydantic and cleaning up the character save function! I must admit the save function wasn't the best but at the time i thought it was ok 😂 Thanks for that and Happy Coding!
